docs/integrations/databases/opentelemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+:::
+
+If you want to create a new Collector, select the **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, we will be configuring the yaml required for Redis Collection.
+
+The log file path configured to capture redis logs must be given here.
+
+The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` file for this information.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
+
+### Step 3: Sending logs to Sumo Logic
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+#### Linux
+
+1. Copy the yaml at **`/etc/otelcol-sumo/conf.d/`** folder in the Redis instance which needs to be monitored.
+2. Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+#### Mac
+
+1. Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2. Restart the otelcol-sumo process using the command `otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"`
+
+Sumo Logic will start receiving data from your host machine when the above command is successfully executed.
+
+This will install the app and dashboards to your Sumo Logic org.
+
+Results will be available after a while. It is important to note that each panel slowly and automatically fills with data matching the time range query received since the panel was created. Your data should appear in the dashboard within 20 minutes, and you will see complete graphs and maps.
+
+## Sample Log Message
+
+Here's a sample log message in Non-Kubernetes environment.
+
+`5275:S 17 Mar 2021  19:13:38.138 * MASTER <-> REPLICA sync: Finished with success`
+
+## Sample Query
+
+This sample Query is from the Redis - Logs dashboard > Logs panel.
+
+```sql
+Query String
```

Comment:
Query string should be outside code block ?

Reply:
@nmohantyctr ☝️